hazelcast/src/main/java/com/hazelcast/query/Predicates.java

```diff
@@ -571,10 +574,25 @@ public static <K, V> PagingPredicate<K, V> pagingPredicate(Predicate<K, V> predi
      * @param target       the target {@link Predicate}
      * @param <K>          the type of keys the predicate operates on.
      * @param <V>          the type of values the predicate operates on.
-     * @throws NullPointerException     if partition key or target predicate is {@code null}
+     * @throws NullPointerException     if partitionKey or target predicate are {@code null}
      */
     public static <K, V> PartitionPredicate<K, V> partitionPredicate(Object partitionKey, Predicate<K, V> target) {
-        return new PartitionPredicateImpl<>(partitionKey, target);
+        checkNotNull(partitionKey, "partitionKey can't be null");
+        return new PartitionPredicateImpl<>(Collections.singleton(partitionKey), target);
     }
 
+    /**
+     * Creates a new partition predicate that restricts the execution of the target predicate to a subset of partitions.
+     *
+     * @param partitionKeys the partition keys
+     * @param target       the target {@link Predicate}
```

Comment:
minor: maybe add one space for the each param after the partitionKeys so that the docs would be in-line